hack/update-release-parameters.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
```

Comment:
♻️ I recommend using `set -e` to bail when any single command fails. If it is okay for something to fail, `|| true` or '|| :` allows it.

♻️ I recommend using `set -u` to bail when an unset variable is used. For example, if someone forgets to pass an argument to the script it will exit non-zero with a message about needing `$1`.

```suggestion
#!/usr/bin/env bash
set -eu

```